torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4537,6 +4537,18 @@ def sample_inputs_narrow_copy(op_info, device, dtype, requires_grad, **kwargs):
                              requires_grad=requires_grad)
         yield SampleInput(tensor, args=args)
 
+# def reference_inputs_narrow_copy(op_info, device, dtype, requires_grad, **kwargs):
```

Comment:
not needed?

Reply:
I left this deliberately to show that sparse coverage is missing. I'll open an issue and link to it instead. Cannot uncomment this test right now because the ref tests will fail.